gradio/components/chatbot.py

```diff
@@ -38,6 +38,8 @@ def __init__(
         | Callable
         | None = None,
         color_map: dict[str, str] | None = None,  # Parameter moved to Chatbot.style()
+        enable_latex: bool = True,
+        latex_delimiters: list[dict[str, str | bool]] | None = [{"left": "$$", "right": "$$", "display": True}],
```

Comment:
Just wondering, do we need the `display` parameter? Confused by what it does

Reply:
I would personally prefer just a list of strings if possible. Like: `["$", "$$"]` to mean that either single dollar signs of double dollar signs get converted to latex

Reply:
I know some delimeters are not symmetric, but in that case, we could just accept the "open" delimeters and take care of everything else ourself. Would be better DX than having to deal with this nested dict